lib/Models/Feature.js

```diff
@@ -92,7 +92,7 @@ Feature.fromEntityCollectionOrEntity = function(entity) {
     if (entity.entityCollection) {
         feature = entity.entityCollection.getById(entity.id);
     }
-    if (!feature || !feature instanceof Feature) {
+    if (!feature || !(feature instanceof Feature)) {
```

Comment:
Oh good catch.  It probably means that we weren't correctly creating a `Feature` for something that started out as just an `Entity` though.  @RacingTadpole is there anything in particular we should test to make sure that doesn't have unintended side effects?